edit_course/course_forms.py

```diff
@@ -91,8 +97,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.fields["assistants"].widget.attrs["class"] = "search-select"
-        self.fields["assistants"].help_text = ""
+        self.fields['assistants'].queryset = self.instance.assistants.values_list('user', flat=True)
+        self.fields['assistants'].widget.attrs["data-users-api-url"] = api_reverse("user-list")
```

Comment:
If the new field class hard codes the User model in it, then the `self.fields['assistants'].widget.attrs["data-users-api-url"] = api_reverse("user-list")` part could also be defined there inside the field class (`self.widget` instead of `self.fields['assistants']`).